### ISSUE TITLE
WRST TW0/TWN in restart write

### ISSUE THREAD

**Issue by JessicaMeixner-NOAA** [NOAA-EMC/WW3#217]
WRST assumes it's writing a restart at TWN, however, if a restart is asked for at initialization time, we could in fact write out TW0.

**Comment by aliabdolali** [comment]
When we define Type 4 output for the restart file, starting at initialization time, it does not write restart000.ww3 by default. Is there an option or switch for it? Does WRST switch invoke this feature?

**Comment by JessicaMeixner-NOAA** [comment]
That's a good point, I think it would still be restart001.ww3 not restart000.ww3.  This was reported by @ajhenrique, I'm guessing he saw this in the second set of restarts where the date YYYYMMDD.HHMMSS.restart.ww3 is written.  I never really understood why you would want to write out a restart file at the beginning time though, so maybe instead there should be a check in the model to not write out the restart at initialization time, instead of the check for TW0 vs TWN.  WRST should have no effect on when a restart is written, it just says write out the wind file in the restart when you write one out.

**Comment by aliabdolali** [comment]
Yes, it is restart001.ww3 and it is created at t=0 but with zero sizes. Once it reaches the first time step for type 4, it will be filled. 
I am not sure, but the restart002.ww3 is created and filled at the second time step.
The restart at t=0 does not make sense except for the check.

**Comment by JessicaMeixner-NOAA** [comment]
What about for this issue, if I  create a check that you do not write a restart file at initialization time?

**Comment by aliabdolali** [comment]
@JessicaMeixner-NOAA Yes, this is what we need to do.

**Comment by JessicaMeixner-NOAA** [comment]
@aliabdolali, my plan is to close this issue as unplanned work.  Any objection?

**Comment by MatthewMasarik-NOAA** [comment]
@aliabdolali @JessicaMeixner-NOAA, can this be closed as unplanned work?
